packages/calling-stateful-client/src/CallClientState.ts

```diff
@@ -696,6 +704,44 @@ export interface IncomingCallState {
   endTime?: Date;
 }
 
+/**
+ * State only version of {@link @azure/communication-calling#TeamsIncomingCall}
+ * @beta
+ */
+export interface TeamsIncomingCallState {
```

Comment:
Extra common part of 2 incomingcall state, looks like the only difference is "info: CallInfo"